app/actions/v3/service_binding_create.rb

```diff
@@ -64,6 +64,26 @@ def save_failed_state(binding, e)
         )
       end
 
+      def complete_binding_and_save(binding, binding_details, last_operation)
+        binding.save_with_attributes_and_new_operation(
+          binding_details.symbolize_keys.slice(*permitted_binding_attributes),
+          {
+            type: 'create',
+            state: last_operation[:state],
+            description: last_operation[:description],
+          }
+        )
+
+        notify(binding)
```

Comment:
I'm wondering if there is a better name for this that would make it even more decoupled from the specific action - but I cannot think of one. Maybe just `notify_diego` might be at least more descriptive

Reply:
maybe something like "post_bind_action" is generic enough to express that this action can be specific and different for each binding

Reply:
yeah, I didn't want to mention `diego` at all. I think `post_bind_action` is a good enough hook name, much better than notify.